src/main/scala/uk/gov/nationalarchives/dp/client/EntityClient.scala

```diff
@@ -629,34 +630,33 @@ object EntityClient {
 
   /** Represents a Preservica security tag
     */
-  sealed trait SecurityTag {
-    override def toString: String = getClass.getSimpleName.dropRight(1).toLowerCase
+  enum SecurityTag:
+    override def toString: String = this match
+      case Open   => "open"
+      case Closed => "closed"
+    case Open, Closed
+
+  object SecurityTag {
+
+    def fromString(securityTagString: String): Option[SecurityTag] = securityTagString match {
+      case "open"   => Option(Open)
+      case "closed" => Option(Closed)
+      case _        => None
+    }
   }
 
   /** Represents an entity type
     */
-  sealed trait EntityType {
-
-    /** The path to be used in the url information-objects, structural-objects or content-objects
-      */
-    val entityPath: String
-
-    /** Either IO, SO or CO
-      */
-    val entityTypeShort: String
-
-    /** A string representing the implementing classes name
-      * @return
-      *   The class name
-      */
-    override def toString: String = getClass.getSimpleName.dropRight(1)
-  }
+  enum EntityType(val entityPath: String, val entityTypeShort: String):
+    case StructuralObject extends EntityType("structural-objects", "SO")
+    case InformationObject extends EntityType("information-objects", "IO")
+    case ContentObject extends EntityType("content-objects", "CO")
 
   /** Represents a Preservica representation tag
     */
-  sealed trait RepresentationType {
-    override def toString: String = getClass.getSimpleName.dropRight(1)
-  }
+
+  enum RepresentationType:
+    case Access, Preservation
```

Comment:
Should we add a `toString` method for this since it was there before?

Reply:
As above and the same for the other enums.